.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -449,13 +651,41 @@ stages:
             echo '##vso[task.setvariable variable=ENABLE_FIPS]false'
             echo '##vso[task.setvariable variable=ENABLE_TRUSTED_LAUNCH]True'
             echo '##vso[task.setvariable variable=SGX_INSTALL]True'
+            echo '##vso[task.setvariable variable=ENABLE_CGROUPV2]False'
           displayName: Setup Build Variables
         - template: ./templates/.builder-release-template.yaml
           parameters:
             artifactName: marinerv2-gen2-trustedlaunch
+  - stage: build_vhd_marinerv2_gen2_trustedlaunch_cgroupv2
+    dependsOn: []
+    condition: eq('${{ parameters.buildMarinerV2gen2TrustedLaunchCgroupV2 }}', true)
+    jobs:
+    - job: build
+      timeoutInMinutes: 180
+      steps:
+        - bash: |
+            echo '##vso[task.setvariable variable=DRY_RUN]${{parameters.dryrun}}'
+            echo '##vso[task.setvariable variable=OS_SKU]CBLMariner'
+            echo '##vso[task.setvariable variable=OS_VERSION]V2'
+            echo '##vso[task.setvariable variable=IMG_PUBLISHER]MicrosoftCBLMariner'
+            echo '##vso[task.setvariable variable=IMG_OFFER]cbl-mariner'
+            echo '##vso[task.setvariable variable=IMG_SKU]cbl-mariner-2-gen2'
+            echo '##vso[task.setvariable variable=HYPERV_GENERATION]V2'
+            echo '##vso[task.setvariable variable=AZURE_VM_SIZE]Standard_DS2_v2'
+            echo '##vso[task.setvariable variable=FEATURE_FLAGS]None'
+            echo '##vso[task.setvariable variable=CONTAINER_RUNTIME]containerd'
+            echo '##vso[task.setvariable variable=ARCHITECTURE]X86_64'
+            echo '##vso[task.setvariable variable=ENABLE_FIPS]false'
+            echo '##vso[task.setvariable variable=ENABLE_TRUSTED_LAUNCH]True'
+            echo '##vso[task.setvariable variable=SGX_INSTALL]True'
+            echo '##vso[task.setvariable variable=ENABLE_CGROUPV2]True'
+          displayName: Setup Build Variables
+        - template: ./templates/.builder-release-template.yaml
+          parameters:
+            artifactName: marinerv2-gen2-trustedlaunch-cgroupv2
   - stage: build_vhd_marinerv2_gen2_kata_trustedlaunch
     dependsOn: []
-    condition: eq('${{ parameters.buildMarinerV2gen2kataTrustedLaunch}}', true)
+    condition: eq('${{ parameters.buildMarinerV2gen2kataTrustedLaunch }}', true)
```

Comment:
nit: trailing space?

Reply:
I noticed a leading space in the front and thus I add a trailing space to match with that.

Reply:
oh hah I missed that